docs/book/src/migration/v3vsv4.md

```diff
@@ -0,0 +1,85 @@
+# go/v3 vs go/v4-alpha 
+
+This document covers all breaking changes when migrating from projects built using the plugin go/v3 (default for any scaffold done since `28 Apr 2021`) to the next alpha version of the Golang plugin `go/v4-alpha`.
+
+The details of all changes (breaking or otherwise) can be found in:
+- [controller-runtime][controller-runtime]
+- [controller-tools][controller-tools]
+- [kustomize][kustomize-release]
+- [kb-releases][kb-releases] release notes.
+
+## Common changes
+
+- `go/v4-alpha` projects use Kustomize v4x (instead of v3x)
+- note that some manifests under `config/` directory changes in order to no longer use the deprecated Kustomize features
+such as env vars. 
+- adds support for Apple Silicon M1 (darwin/arm64)
+- remove support to CRD/WebHooks Kubernetes API v1beta1 version which are no longer supported since k8s 1.22
+- no longer scaffold webhook test files with `"k8s.io/api/admission/v1beta1"` the k8s API which is no longer served since k8s `1.25`. By default
+webhooks test files are scaffolding using `"k8s.io/api/admission/v1"` which is support from k8s `1.20`
+- no longer provide backwards compatible support with k8s versions < `1.16`
+
+<aside class="note">
+<H1> TL;DR of the New `go/v4-alpha` Plugin </H1>
```

Comment:
See that below contains another `tldr` header with contents. Do we need this `h1` block?

Reply:
All notes have that. It creates a title @Kavinjsir